Sources/Foundation/Extensions/URLRequest.UserAgent.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  URLRequest.UserAgent.swift
```

Comment:
The convention we've used is to name the file `<module>.<typename>.swift`. I.e. this should be named `Foundation.URLRequest.swift`. This is to avoid file name collisions when Cocoapods merges all the files from the various modules into a single module.